manifests/profile/vim.pp

```diff
@@ -9,7 +9,7 @@
 # @example
 #   include nebula::profile::vim
 class nebula::profile::vim {
-  package { 'vim': }
+  package { ['vim', 'vim-gtk', 'vim-nox': }
```

Comment:
vim-gtk is going to bring in X dependencies everywhere, which we probably don't want.